routes/auth-routes.js

```diff
@@ -28,4 +29,6 @@ router.post('/notebooks(/*)', authToken, upload, function(req, res) {
     res.status(201).json(resBody);
 });
 
+router.delete('/notebooks(/*)', authToken, nbdelete);
```

Comment:
Maybe a short comment like the one on post() for new folks who might read it as taking some delete action immediately when called.